index.html

```diff
@@ -0,0 +1,857 @@
+
+          <div class="news">
+            <div class="news-img">
+              <img src="./images/l-3.jpg" alt="latest news 3 image" />
+            </div>
+            <div class="news-text-author">
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+                neque!
+              </p>
+              <span>Author1 10 hours ago</span>
+            </div>
+          </div>
+
+          <div class="news">
+            <div class="news-img">
+              <img src="./images/l-4.jpg" alt="latest news 4 image" />
+            </div>
+            <div class="news-text-author">
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+                neque!
+              </p>
+              <span>Author1 10 hours ago</span>
+            </div>
+          </div>
+
+          <div class="news">
+            <div class="news-img">
+              <img src="./images/l-5.jpg" alt="latest news 5 image" />
+            </div>
+            <div class="news-text-author">
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+                neque!
+              </p>
+              <span>Author1 10 hours ago</span>
+            </div>
+          </div>
+
+          <div class="news">
+            <div class="news-img">
+              <img src="./images/l-6.jpg" alt="latest news 6 image" />
+            </div>
+            <div class="news-text-author">
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+                neque!
+              </p>
+              <span>Author1 10 hours ago</span>
+            </div>
+          </div>
+
+          <div class="news">
+            <div class="news-img">
+              <img src="./images/l-3.jpg" alt="latest news 7 image" />
+            </div>
+            <div class="news-text-author">
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+                neque!
+              </p>
+              <span>Author1 10 hours ago</span>
+            </div>
+          </div>
+
+          <div class="news">
+            <div class="news-img">
+              <img src="./images/l-2.jpg" alt="latest news 8 image" />
+            </div>
+            <div class="news-text-author">
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+                neque!
+              </p>
+              <span>Author1 10 hours ago</span>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- 
+      ======================================================================================
+      Funding Section
+      ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+
+      <section class="funding">
+        <h2 class="heading2 orange">Latest Funding Rounds on index.co ></h2>
```

Comment:
- [ ] Kindly fix the text layout for big and medium devices.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83334076-a4e5a280-a269-11ea-8d23-389b407ccd4e.png) | ![image](https://user-images.githubusercontent.com/57414789/83334071-98614a00-a269-11ea-877e-71ada8136a41.png)